src/kirby/components/segmented-control/segmented-control.component.tns.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../../kirby/scss/native-imports";
+
+.segment-control {
+  border-radius: 20;
+  background: get-color("kirby-white");
+  color: get-color("kirby-grey-9");
+  display: inline-block;
+  height: size("xl");
+  margin-top: size("s");
+  margin-bottom: size("s");
+}
+
+.segment-item {
+  font-weight: font-weight("normal");
+  font-size: font-size("s");
+  padding-left: size("m");
+  padding-right: size("m");
+  border-radius: 20;
+  height: 100%;
+
+  &.is-checked {
+    color: get-color("kirby-white");
+    background-color: get-color("kirby-grey-9");
+  }
+
+  & + & {
+    margin-left: size("xs") * -1;
+  }
+}
+
+.segment-badge-wrapper {
+  margin-bottom: 30;
+  z-index: 2;
```

Comment:
Consider creating a central place for z-indexes... having their values inlined "all over the place" can become quite messy over time (eg. a modal z-index offset may clash with this)